### PR TITLE
Update CIFAR10 example

### PR DESCRIPTION
**Pull request by AlexeyG**
* Added TB.dev logs for 8x and 1x V100 GPU runs
* Added corresponding wall time
* Nit changes to the training scripts
* Re-structured the README

### CODE REVIEW

File: examples/cifar10/README.md [hunk 1]
```diff
@@ -1,142 +1,52 @@
...
+
+#### Wide ResNet: 26 layers, 10x width (Zagoruyko and Komodakis, 2017)
+| Hardware | Epochs | Learning rate | Training time | Error rate | TensorBoard.dev |
+| --- | --- | --- | --- | --- | --- |
+| 1 x Nvidia V100 (16GB) | 200 | Piece-wise constant | 4h 36m | 4.45% | [2020-03-22](https://tensorboard.dev/experiment/1kE2bq9RR7WrT7Zw14fNpg/) |
+| 8 x Nvidia V100 (16GB) | 200 | Piece-wise constant | 57m | 3.93% | [2020-03-22](https://tensorboard.dev/experiment/IpzU0txnR7uZnExGmk7IEA/) |
+
+#### Wide ResNet: 26 layers, 6x width, Shake-Shake regularization (Gastaldi, 2017)
+| Hardware | Epochs | Learning rate | Training time | Error rate | TensorBoard.dev |
+| 1 x Nvidia V100 (16GB) | 200 | Piece-wise constant | 3h 38m | 3.43% | [2020-03-22](https://tensorboard.dev/experiment/dJc9e4k1R5mC0DPVZyYrpg/) |
```
Comment (avital) [inline examples/cifar10/README.md]: These don't render as tables for some reason (also they don't have the row with --- | ---, maybe that's related?)

File: examples/cifar10/README.md [hunk 2]
```diff
@@ -1,142 +1,52 @@
...
+| 8 x Nvidia V100 (16GB) | 200 | Piece-wise constant | 54m | 3.39% | [2020-03-26](https://tensorboard.dev/experiment/l2CPAqpnTlCjjZKbnuOomg/) |
+| 1 x Nvidia V100 (16GB) | 1800 | Cosine | 1d 9h 25m | 2.97% | [2020-03-22](https://tensorboard.dev/experiment/aXyhYX2oSxKH5lIlj4Ao5g/) |
+| 8 x Nvidia V100 (16GB) | 1800 | Cosine | 8h 5m | 2.82% | [2020-03-26](https://tensorboard.dev/experiment/kq303ri1RHygWxb5YrLp1Q/) |
+
+#### PyramidNet, Shake-drop regularization (Han *et al.*, 2017; Yamada *et al.*, 2018)
+| 8 x Nvidia V100 (16GB) | 300 | Piece-wise constant | 6h 41m | 3.25% | [2020-03-24](https://tensorboard.dev/experiment/OpZLDnVjRhmJKXq62RoaUQ/) |
```
Comment (avital) [inline examples/cifar10/README.md]: These don't render as tables for some reason (also they don't have the row with --- | ---, maybe that's related?)

Comment (AlexeyG) [reply examples/cifar10/README.md]: Fixed. Thanks for spotting this!

File: examples/cifar10/models/utils.py [hunk 3]
```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-"""Shake-shake and ShakeDrop functions."""
+"""Shake-shake and dhake-drop functions."""
```
Comment (avital) [inline examples/cifar10/models/utils.py]:
```suggestion
"""Shake-shake and shake-drop functions."""
```

**Pull request merged**